docs/doc_sources/contributor_guides/building.rst

```diff
@@ -3,18 +3,20 @@
 Building from the Source
 ========================
 
-To build :py:mod:`dpctl` from the source, you need DPC++ compiler.
+To build :py:mod:`dpctl` from the source, you need the Intel(R) oneAPI DPC++ compiler.
 To run examples and test suite you would need GPU drivers and/or CPU
 OpenCL drivers. It is preferable to use the Intel(R) oneAPI DPC++ compiler
-available as part of oneAPI Base-Kit. However, it is possible to use a custom
+available as part of the oneAPI Base Kit. However, it is possible to use a custom
 build of DPC++ to build :py:mod:`dpctl`, especially if you want to enable
-CUDA support or try latest features.
+CUDA* support or try the latest features.
 
 Building using oneAPI DPC++
 ---------------------------
 
-Install oneAPI and graphics drivers to the system prior
-to proceeding further.
+prerequisites
```

Comment:
@oleksandr-pavlyk which drivers does a user need? Is there a site we can point them to? And what does it mean to "Install oneAPI"? Should the user install a specific toolkit? Or just the DPC++ compiler?

Reply:
This part needs to be expanded indeed.  The needed drivers would depend on the user's hardware. If the user intends to target Intel GPUs, drivers should be installed following https://dgpu-docs.intel.com/driver/installation.html as appropriate for the hardware configuration. 

If the user intends to target NVidia, or AMD GPU cards, vendor drivers are required, in addition to CodePlay plugins for the installed version of oneAPI DPC++ compiler.   

If the user wants to target CPU device, OpenCL CPU driver comes as a part of oneAPI DPC++ installation, and should be available out of the box. The CPU driver is also packaged in conda, and is automatically made available using conda activation scripts on Linux, and on Windows (in user-mode). If conda is used by a user with elevated privileges (think GitHub Actions CI), additional steps is required of running a PowerShell script available in ``${CONDA_PREFIX}\Scripts\`. Use script `set-intel-ocl-icd-registry.ps1` to set appropriate registry key, and `unset-intel-ocl-icd-registry.ps1` to remove it.

Reply:
@oleksandr-pavlyk please review my new commit. I added content based on your comments.